tui-markdown/src/lib.rs

```diff
@@ -36,16 +36,24 @@
     unused_variables,
     unused_imports,
     unused_mut,
-    unused_assignments
+    unused_assignments,
 )]
 
 use std::{sync::LazyLock, vec};
 
+// Public-facing theming and configuration APIs --------------------------------
+mod style_sheet;
+pub use style_sheet::{BuiltinStyleSheet, DefaultStyleSheet, StyleSheet};
```

Comment:
The Builtin vs Default distinction is a little confusing. Can you expand on why this is necessary / what it solves? Without understanding this, I'd intuitively suggest perhaps keep only one of these.